### ISSUE TITLE
more colors in textual output

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#2500]
There is a red warning and 3 white Warning, and the Info Text is orange

![image](https://github.com/user-attachments/assets/5f659ffc-ba29-42b8-81b7-61a5c1bc4564)

I think we should have all warnings in the same color, different to info but not white, as in this example

**Comment by anshulagrawal2902** [comment]
I agree with you, I also noticed that the RED warning is actually an INFO statement from the application's point of view. I am interested in taking a look, could you assign this issue to me ?

**Comment by anshulagrawal2902** [comment]
I was looking into this and I don't understand why are we not using a `singleton` logger instance. I saw definition for the `setup_logger` function, but we rarely use it. Most modules have an `import logging` statement and they are using the root logger. This makes it difficult to make changes like the one in this issue. Either I misunderstood it or I am lacking context. Regardless, I would appreciate your view on the problem.

**Comment by ReimarBauer** [comment]
The whole thing has simply grown over 10 years and currently most of the tasks for MSS are to refactor it and increase stability.
We have to make sure that we can e.g. suppress the matplotlib logger and output all our logging messages. That is much easier by a singleton and logging definitions so that different users can use their own.
